docs/content/guide/es/06_variable-replacement.ngdoc

```diff
@@ -89,7 +89,7 @@ angular.module('myApp').controller('Ctrl', ['$scope', function ($scope) {
 y recién después pasarlos como expresión a través del filtro:
 
 <pre>
-{{ 'TRANSLATION_ID' | translate:translationData }}
```

Comment:
Exactly. I fully agree with you. Don't start "localizing" the technical references. It should be identical across all languages.